apps/server/src/modules/deletion/controller/dto/deletion-execution.params.spec.ts

```diff
@@ -0,0 +1,38 @@
+import { DeletionExecutionParams } from './deletion-execution.params';
+
+describe(DeletionExecutionParams.name, () => {
+	describe('constructor', () => {
+		describe('when passed properties', () => {
+			let deletionExecutionParams: DeletionExecutionParams;
+
+			beforeEach(() => {
```

Comment:
To make it consistence with the rest of the stack please use a setup method.
```
const setup = () => {
     const deletionExecutionParams = new DeletionExecutionParams();
     
     return { deletionExecutionParams  }
}
```

Reply:
done